pkg/reconciler/workload/syncer/pushmanager.go

```diff
@@ -91,7 +91,6 @@ func (m *pushSyncerManager) update(ctx context.Context, cluster *workloadv1alpha
 	}
 
 	klog.Infof("Started push mode syncer from clusterName %s for pcluster %s", kcpClusterName, cluster.Name)
-	conditions.MarkTrue(cluster, workloadv1alpha1.WorkloadClusterReadyCondition)
```

Comment:
We don't need SyncerReady=True here?

Reply:
Yep, not sure why I removed it. `checkHealth` doesn't get called in the func that calls update if an update was needed.